app/components/ActionButtons.tsx

```diff
@@ -40,9 +40,18 @@ interface Props {
   onDelete?: Function;
   defaultActiveColor?: string;
   children?: any;
+  editIconStyle?: any;
+  delIconStyle?: any;
 }
 
-export default function Actionbuttons({ request, onDelete, defaultActiveColor, children }: Props) {
+export default function Actionbuttons({
+  request,
+  onDelete,
+  defaultActiveColor,
+  children,
+  editIconStyle = '',
```

Comment:
let's set default value to empty object.

Reply:
set with `{}`